strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/domain/ArtifactEntry.java

```diff
@@ -31,6 +31,12 @@
      */
     private String artifactPath;
 
+    private Long sizeInBytes;
```

Comment:
@fuss86 @carlspring 
guys, I think this is related to [SB-900](https://youtrack.carlspring.org/issue/SB-900). I mean that we need some kind of separated `ArtifactAttributes` entity to store various attributes like here and like attributes from `SB-900`.
So maybe we should do it here and now? :)

Reply:
@sbespalov , @fuss86 ,

If I recall correctly, the consensus for this was for it to be handled as a separate task, right?

Reply:
Yes, we need to handle SB-930 one day. I've created a PR confirming with have got an issue in orientDb with cascade updates here: https://github.com/strongbox/strongbox/pull/384

Reply:
Okay. Let's handle it separately.